SConstruct

```diff
@@ -943,6 +943,8 @@ if "SAVE_OPTIONS" in ARGUMENTS :
 
 if env["BUILD_CACHEDIR"] != "" :
 	CacheDir( env["BUILD_CACHEDIR"] )
+else:
```

Comment:
It still feel like this bit belongs in the IE config. If you haven't enabled the build cache, you don't need to receive a message telling you it has been disabled...

I'm off home now I hope...feel free to merge after addressing this (or not if the consensus is it makes sense this way)...